R/pkg/R/generics.R

```diff
@@ -919,10 +920,9 @@ setGeneric("array_contains", function(x, value) { standardGeneric("array_contain
 #' @export
 setGeneric("ascii", function(x) { standardGeneric("ascii") })
 
-#' @param x Column to compute on or a GroupedData object.
```

Comment:
I think we need to add this back to GroupedData, since that is now in a different rd?